components/Input/Input.jsx

```diff
@@ -64,37 +25,31 @@ const InputIcon = styled(Icon)`
   bottom: 10px;
 `;
 
-const InputSearchIcon = styled(InputIcon)`
+const InputSearchIcon = styled(InputIcon).attrs({ name: 'search' })`
```

Comment:
Why did you need this `name: 'search'`?

Reply:
Because it's an Icon component, so it needs this prop to render a "search" one. :)